### ISSUE TITLE
Also search hidden files?

### ISSUE THREAD

**Issue by trusktr** [cloudhead/neovim-fuzzy#30]
It seems to ignore hidden files by default. How can we make it also include hidden results? - Joe

**Comment by ohjames** [comment]
Yeah this is also bothering me.

**Comment by cloudhead** [comment]
I'll add an option for this, it's trivial.

**Comment by christophweegen** [comment]
Any update on this? Thanks!

**Comment by cloudhead** [comment]
Well, I thought about it more; but the thing is this will search into your `.git` folder too, which will be pretty annoying, no? Can you explain your use-case, maybe I can come up with a nice solution if I understand the problem.

**Comment by christophweegen** [comment]
@cloudhead thanks for your answer! 

One solution would be we could filter all paths that have `.git` as a path segment via regex. Not sure about performance in big directories, but this would solve the problem of having `.git` files in search results. 

About performance: I also use fzy at the command line to switch to directories fastly. Since listing all the directories on your machine needs some seconds, I save them to a cache file where fzy can get its paths from instantly. Works like a charm. I just need to update this file from now and then if new directories were added.

Below are two bash functions I use for that. `fb` (mnemonic for "fzy build") builds a file with all dir path found in your home dir. This needs some seconds, but can be sped up a lot by ripgrep. Just added it with standard bash functions for easy try out. `f` pipes these file paths into `fzy`. This happens instantly and let you switch to a dir of your choice:

```
fb(){
  cd && find . -type d > ~/fuzzy.cache
}

f(){
  cd "$(cat ~/fuzzy.cache | fzy)"
}
```

Maybe this could give you some inspiration in case you run into performance problems while filtering unwanted file paths.

Thanks and best wishes

**Comment by cloudhead** [comment]
Fixed in #49
